src/components/composites/NumberInput/NumberInputStepper.tsx

```diff
@@ -32,6 +32,13 @@ export const NBStepper = ({ children, ...props }: any) => {
         {...(disablitityCheck || isDisabled ? _disabled : {})}
         borderColor="transparent"
         style={style}
+        opacity={disablitityCheck || isDisabled ? 0.4 : 1}
+        {...(Platform.OS === 'web'
+          ? {
+              disabled: disablitityCheck || isDisabled,
```

Comment:
We have already added it in line 32. Why are we adding it again?

Reply:
Adding disable on make the component unfit for `focus`. That's why adding disabled only for the web.